app/controllers/feedbacks_controller.rb

```diff
@@ -43,9 +44,11 @@ def show
   end
 
   def edit
-    @submissions = apply_scopes(Submission)
+    @submissions = Submission
```

Comment:
```suggestion
    @submissions = Submission.all
```
This should at least by a activerecord query, IMO. Right now it's just referring to the class, which seems counter-intuitive.